src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -1085,7 +1091,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       return false
     }
 
-    if (isRequirementMet && !isSkin && !isSmartWearable && modelSize && modelSize > MAX_WEARABLE_FILE_SIZE) {
+    if (isRequirementMet && !isSkin && !isSmartWearable && finalSize && !isWearableFileSizeValid(finalSize)) {
```

Comment:
are we missing the !isEmote here?
```suggestion
    if (isRequirementMet && !isSkin && !isSmartWearable && !isEmote && finalSize && !isWearableFileSizeValid(finalSize)) {
```